sample.tf

```diff
@@ -2,7 +2,15 @@ terraform {
   required_providers {
     pipeline = {
       source  = "registry.terraform.io/jfrog/pipeline"
-      version = "0.1.0"
+      version = "0.1.1"
+    }
+    project = {
+      source  = "registry.terraform.io/jfrog/project"
+      version = "1.1.0"
+    }
+    artifactory = {
+      source  = "registry.terraform.io/jfrog/artifactory"
+      version = "6.7.1"
```

Comment:
do we want to have other providers here? We don't do this for other projects.

Reply:
We don't currently but we should do the same for Xray and Project providers as well